src/Toggler.js

```diff
@@ -43,10 +46,13 @@ class Toggler extends State {
 
 	/**
 	* Manually expand the content's visibility.
-	* @param {!Element} header
+	* @param {string|!Element} header
 	*/
 	expand(header = this.header) {
-		var content = this.getContentElement_(header);
+		let content = this.getContentElement_(header);
+		if (!core.isElement(header)) {
+			header = this.getHeaderElements_(header);
+		}
 		dom.addClasses(content, this.expandedClasses);
```

Comment:
Hi @fernandosouza , I don't believe that the `addClasses` method and `removeClasses` method can take in a list of elements. As a list of elements will be returned from `querySelectorAll` in `getHeaderElements_`